### DISCUSSION TITLE
请教下如何查看redis 的命中率，进入docker 执行 redis-cli 报错，提示端口不对；

### DISCUSSION THREAD

**Discussion by gukey** [kkkgo/PaoPaoDNS#18] [问答和帮助] [answered]
/ # redis-cli
Could not connect to Redis at 127.0.0.1:6379: Connection refused

**Comment by kkkgo** [comment, marked as answer]
redis被配置为更高效的Unix Socket通信而不是tcp端口，你可以使用以下命令连接：
```
redis-cli -s /tmp/redis.sock
```
另外参考：https://github.com/kkkgo/PaoPaoDNS/issues/5

**Reply by gukey** [reply]
哇，太棒了，谢谢！
